### PR TITLE
chore(readme): Explain transifex specificity of this repository

### PR DESCRIPTION
**Pull request by trollepierre**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,16 +149,21 @@ Community
...
 
-As a _developer_, you must [configure the transifex client][tx-client], and claim an access as _maintainer_ to the [app repository][tx-app]. Then please **only update** the source locale file (usually `en.json` in client and/or server parts), and push it to Transifex repository using the `tx push -s` command.
+As a _developer_, you must [configure the transifex client][tx-client], and claim access as _maintainer_ to the [app repository][tx-app]. Then please **only update** the source locale file (usually `en.json` in client and/or server parts), and push it to Transifex repository using the `tx push -s` command.
 
+###### Drive specificity
+
+CozyDrive and CozyPhotos share this mono-repository but the translations are split into two projects Transifex.
+That is why the translation files are not present in this repository.
+
+To update the translation in other language, make the changes directly on [tx-translate].
```
Comment (Ldoppea) [inline README.md:161]: Your links redirects to the `fr` translation, do we have a "more global" link?

File: README.md [hunk 2]
```diff
@@ -149,16 +149,21 @@ Community
...
+As a _translator_, you can login to [Transifex][tx-signin] (using your Github account) and claim access to the [app repository][tx-app]. Locales are pulled [by the pipeline][yarn tx in travis.yml] when app is build before publishing.
 
-As a _developer_, you must [configure the transifex client][tx-client], and claim an access as _maintainer_ to the [app repository][tx-app]. Then please **only update** the source locale file (usually `en.json` in client and/or server parts), and push it to Transifex repository using the `tx push -s` command.
+As a _developer_, you must [configure the transifex client][tx-client], and claim access as _maintainer_ to the [app repository][tx-app]. Then please **only update** the source locale file (usually `en.json` in client and/or server parts), and push it to Transifex repository using the `tx push -s` command.
 
+###### Drive specificity
+
+CozyDrive and CozyPhotos share this mono-repository but the translations are split into two projects Transifex.
+That is why the translation files are not present in this repository.
```
Comment (Ldoppea) [inline README.md:159]: Is that the real reason? I thought it was just due to the project history and we did not make the change on this repo yet.

Comment (trollepierre) [reply README.md:159]: https://mattermost.cozycloud.cc/cozycloud/pl/oiwy1jcxdbd9f8j3qpruhfzyqc

File: README.md (hunk 1 above)
Comment (trollepierre) [reply README.md]: I found a solution to make it more global

**Review by Ldoppea — APPROVED** [review]

**Pull request merged**